ideascube/library/views.py

```diff
@@ -13,10 +13,10 @@
 from django.views.generic import (CreateView, DeleteView, DetailView, FormView,
                                   ListView, UpdateView, View)
 
+from ideascube.configuration.models import Configuration
```

Comment:
Hmmm...

Abstracting the underlying ORM/model API is a conscious design choice, because I want to deemphasize the usual query API in favour of the `get`/`set` custom API which handles the verifications.

Your suggestion goes even further that way. I like it.